### PR TITLE
[SYSDM] Improve license dialog icon

### PR DESCRIPTION
**Pull request by TAN-Gaming**
## Proposed changes
- Use `mimetypes/application-certificate` icon from Tango Icon Theme 0.8.90 instead of `IDI_CPLSYSTEM`

#### Also:
- Set large icon correctly
- Add NULL check for `DestroyIcon`

Capture

VirtualBox_ReactOS_13_10_2022_14_41_36

### CODE REVIEW

**Review by katahiromz — APPROVED** [review]
LGTM.

**Comment by HBelusca** [comment]
Does the Tango icon set have a "papyrus"-like (document scroll) icon as well?

File: dll/cpl/sysdm/licence.c [hunk 1]
```diff
@@ -23,16 +24,23 @@ OnInitDialog(HWND hDlg, PLIC_CONTEXT pLicInfo)
...
-                                MAKEINTRESOURCE(IDI_CPLSYSTEM),
+                                MAKEINTRESOURCE(IDI_LICENSE),
                                 IMAGE_ICON,
-                                16,
-                                16,
-                                0);
+                                GetSystemMetrics(SM_CXICON),
+                                GetSystemMetrics(SM_CYICON),
+                                LR_DEFAULTCOLOR);
+    pLicInfo->hIconSm = LoadImage(hApplet,
```
Comment (HBelusca) [inline dll/cpl/sysdm/licence.c:32]: How does it look like, if only hIcon and in the SendMessage call below, only ICON_BIG is set?

**Comment by TAN-Gaming** [comment]
> Does the Tango icon set have a "papyrus"-like (document scroll) icon as well?

In version 0.8.90, no.

File: dll/cpl/sysdm/licence.c (hunk 1 above)
Comment (TAN-Gaming) [reply dll/cpl/sysdm/licence.c:32]: VirtualBox_ReactOS_DBG_13_10_2022_21_16_17

The icon is scaled so setting icon manually looks better.

File: dll/cpl/sysdm/licence.c [hunk 2]
```diff
@@ -23,16 +24,23 @@ OnInitDialog(HWND hDlg, PLIC_CONTEXT pLicInfo)
...
+                                  IMAGE_ICON,
+                                  GetSystemMetrics(SM_CXSMICON),
+                                  GetSystemMetrics(SM_CYSMICON),
+                                  LR_DEFAULTCOLOR);
 
     SendMessage(hDlg,
                 WM_SETICON,
-                ICON_SMALL,
+                ICON_BIG,
                 (LPARAM)pLicInfo->hIcon);
```
Comment (HBelusca) [inline dll/cpl/sysdm/licence.c:42]:
```suggestion
    SendMessage(hDlg, WM_SETICON, ICON_BIG, (LPARAM)pLicInfo->hIcon);
```

**Review by HBelusca — APPROVED** [review]

File: dll/cpl/sysdm/licence.c (hunk 2 above)
Comment (TAN-Gaming) [reply dll/cpl/sysdm/licence.c]: Done.

**Review by binarymaster — APPROVED** [review]

**Pull request merged**